src/scripts/main.js

```diff
@@ -1,28 +1,46 @@
 'use strict';
 
-const food = {
-  'Drink': {
-    'Wine': {},
-    'Schnaps': {},
-  },
-
-  'Fruit': {
-    'Red': {
-      'Cherry': {},
-      'Strawberry': {},
+document.addEventListener('DOMContentLoaded', function() {
+  const food = {
+    'Drink': {
+      'Wine': {},
+      'Schnaps': {},
```

Comment:
I don't think it's a good idea to store this variable inside eventListener. Just move it into the upper scope like it was initially